src/pudl/output/eia860.py

```diff
@@ -124,16 +124,25 @@ def plants_eia860(pudl_engine, start_date=None, end_date=None):
         plants_g_eia_tbl.c.plant_id_pudl,
     )
     plants_g_eia_df = pd.read_sql(plants_g_eia_select, pudl_engine)
-
-    out_df = pd.merge(plants_eia_df, plants_eia860_df, how="left", on=["plant_id_eia"])
-    out_df = pd.merge(out_df, plants_g_eia_df, how="left", on=["plant_id_eia"])
+    # generator table for capacity
+    gens_eia860_tbl = pt["generators_eia860"]
+    gens_eia860_select = sa.sql.select(gens_eia860_tbl)
+    plant_capacity = (
+        pd.read_sql(gens_eia860_select, pudl_engine)
+        .groupby(["plant_id_eia", "report_date"], as_index=False)[["capacity_mw"]]
+        .sum(min_count=1)
+        .pipe(apply_pudl_dtypes, group="eia")
+    )
```

Comment:
I'm adding plant capacity in here bc a) i'd like it for the missing plants/manual labeling process but also this is a thing that I have done in a notebook SO MANY times

Reply:
Does the operational status of the generators make this weird? Like will this sum up the capacities of all generators in a plant regardless of whether they're currently in service?